docs: Warn about Codespaces (#6321)
ddev/ddev: docs/content/users/install/ddev-installation.md

    ## GitHub Codespaces

    ??? "Codespaces may not work predictably; Gitpod is more reliable"

>         Unfortunately "your mileage may vary" with Codespaces. It seems to change without warning, and we haven't been able to adequately keep it supported. See [various](https://github.com/ddev/ddev/issues/6102) [issues](https://github.com/ddev/ddev/pull/6101) and [Matthias Andrasch notes](https://my-ddev-lab.mandrasch.eu/tutorials/cloud/codespaces.html).

Reviewer: ```suggestion
        Unfortunately "your mileage may vary" with Codespaces. and we haven't been able to adequately keep it supported. See [various](https://github.com/ddev/ddev/issues/6102) [issues](https://github.com/ddev/ddev/pull/6101) as well as [postCreateCommand issues](https://github.com/devcontainers/features/issues/977).
```
